client/src/components/ReviewCardContainer.js

```diff
@@ -1,18 +1,17 @@
-import React, {useEffect, useState} from 'react';
+import React, {useEffect} from 'react';
 import axios from 'axios';
 import ReviewCard from './ReviewCard';
-
+import {useGlobalState} from '../context/GlobalState';
 
 const ReviewCardContainer = (props) => {
-    const [state, setState] = useState({reviewCards: []})
-
+    const [state, setState] = useGlobalState({reviewCards: []});
+    
     useEffect(async () => {
         await axios.get(`${window.location.origin.toString()}/review/getAllReviews`, {})
         .then(res => {
             console.log(res.data.data);
             let allReviews = [];
-            res.data.data.forEach(x => allReviews.push(<ReviewCard heading={x['User']['UserLogin']} brandName={x['Product']['ProductBrand']} productName={x['Product']['ProductName']} review={x['ReviewText']}/>));
-            console.log(allReviews);
+            res.data.data.forEach(x => allReviews.push(<ReviewCard heading={`${x['User']['UserLogin']}, ${(new Date(x['ReviwedAt'])).toLocaleString('default', { month: 'short', day: 'numeric', year: 'numeric'})} at ${(new Date(x['ReviwedAt'])).toLocaleString('default', { timeStyle: 'long'})}`} brandName={x['Product']['ProductBrand']} productName={x['Product']['ProductName']} review={x['ReviewText']}/>));
```

Comment:
Can we make a separate function for the creating the Data. Something such as createDate function ?